int16 BatteryStatus 
int16 BatteryVoltage
float64 Percentage
int16 ChargingCurrent
bool ChargingFlag
int16 Temperature
int16 RemainCapacity
int16 FullCapacity
